po/pl.po

```diff
@@ -101,23 +99,23 @@ msgid "(unknown)"
 msgstr "(nieznane)"
 
 #: src/gpodder/util.py:1372 src/gpodder/util.py:1394
-#, python-format
+#, fuzzy, python-format
```

Comment:
fuzzy translations will be ignored at runtime: you want to remove them!

Reply:
Oh, I see what happened: Poedit complained about *\r* signs at the end of every translation with different plural versions. So it tagged them as fuzzy. I don't really know if it something to worry about so I just set them as checked, because they are translated well.

Reply:
I think they're fine with "\r" even if msgmerge and poedit complain.